docs/setup/docker.asciidoc

```diff
@@ -35,10 +35,16 @@ ifeval::["{release-state}"!="unreleased"]
 docker pull {docker-repo}:{version}
 --------------------------------------------
 
-Alternatively, you can download other Docker images that contain only features 
-available under the Apache 2.0 license. To download the images, go to 
+Alternatively, you can download other Docker images that contain only features
+available under the Apache 2.0 license. To download the images, go to
 https://www.docker.elastic.co[www.docker.elastic.co].
 
+=== Running Kibana on Docker for development
+Kibana can be quickly started and connected to a local Elasticsearch container for development
+or testing use with the following command:
+--------------------------------------------
+docker run --net=host -e "ELASTICSEARCH_HOSTS=http://localhost:9200" {docker-repo}:{version}
```

Comment:
```suggestion
docker run --link {elasticsearch-container-name-or-id}:elasticsearch -p 5601:5601 {docker-repo}:{version}
```

I was trying this locally and after that I don't think this is the best command to list here as help.
I think we should use `docker run --link {elasticsearch-container-name-or-id}:elasticsearch -p 5601:5601 {docker-repo}:{version}` instead. The one you used didn't work on macOS when you ran elasticsearch docker for development with the recommended way on the current docs: `docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.6.1
`

Can you also change that @pgoldtho please?

Reply:
Interesting  `-e "ELASTICSEARCH_URL=http://localhost:9200"` worked on Ubuntu but not on OS-X